src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/AbstractReferenceFinder.cs

```diff
@@ -109,6 +103,22 @@ protected static async Task<ImmutableArray<Document>> FindDocumentsAsync(Project
             }, cancellationToken);
         }
 
+        /// <summary>
+        /// Finds all the documents in the provided project that contain the requested string
+        /// values
```

Comment:
"requested string values" 

What requested string values? This just finds where `ContainsGlobalAttribues` is true